src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -240,6 +240,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('vnet_name', help='Name of a virtual network.', validator=validate_subnet)
         c.argument('action', help='The action of virtual network rule.')
 
+    with self.argument_context('storage account blob-service-properties show') as c:
+        c.argument('account_name', acct_name_type, id_part=None)
+
+    with self.argument_context('storage account blob-service-properties update') as c:
+        c.argument('account_name', acct_name_type, id_part=None)
+        c.argument('enable_change_feed', arg_type=get_three_state_flag(),
```

Comment:
Please give more detailed help message for ChangeFeed feature and use active voice.

Reply:
Add help in _help.py

---
In reply to: [354645627](https://github.com/Azure/azure-cli/pull/11488#discussion_r354645627) [](ancestors = 354645627)